src/main/java/com/sda10/carrental/service/CarService.java

```diff
@@ -34,9 +38,26 @@ public Car updateCar(Long id, Car car){
         }
     }
 
-    public void deleteCar(Long id){
-        Car existingCar=carRepository.findById(id).get();
+    public void deleteCar(Long id) {
+        Car existingCar = carRepository.findById(id).get();
 
         carRepository.delete(existingCar);
     }
+
+    public List<Car> carFilters() {
```

Comment:
Aici ar trebui sa ai un parametru in care sa primesti valorile dupa care faci cautarea (filtrarea)